sap/xssec/security_context_xsuaa.py

```diff
@@ -118,17 +118,13 @@ def _init_xsappname(self):
                                      ' the manifest.yml (legacy) as well as in xs-security.json.'
                                      ' Remove it in manifest.yml.')
 
-    def _validate_jku(self):
-        # configured uaa domain must be part of jku in order to trust jku
-        uaa_domain = self._config['uaadomain']
-
+    def _get_jku(self):
+        uaa_domain: str = self._config.get('uaadomain') or self._config.get('url')
         if not uaa_domain:
             raise RuntimeError("Service is not properly configured in 'VCAP_SERVICES'")
-
-        jku_url = urllib3.util.parse_url(self._properties['jku'])
-        if not jku_url.hostname.endswith(uaa_domain):
-            self._logger.error("Error: Do not trust jku '{}' because it does not match uaa domain".format(self._properties['jku']))
-            raise RuntimeError("JKU of token is not trusted")
+        uaa_domain = re.sub(r'https://', '', uaa_domain)
```

Comment:
@allenlkq This is the same as:
`f'https://{uaa_domain}'`
is it? if it's the case, why you decided to use `re`?

Reply:
it's to remove `https://` from the `uaa_domain` in case it's there.

Reply:
I was planning to do a more strict matching where only `https://` at the beginning of the domain is replaced. I missed `^` in the regex.